client/src/state/history/revertToBlockVersionMutation.js

```diff
@@ -0,0 +1,43 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const mutation = gql`
+mutation revertBlockToVersion($id:ID!, $fromStage:VersionedStage!, $toStage:VersionedStage!, $fromVersion:Int!) {
+  copyBlockToStage(Input: {
+    ID: $id
+    FromVersion: $fromVersion
+    FromStage: $fromStage
+    ToStage: $toStage
+  }) {
+    ID
+  }
+}
+`;
+
+const config = {
+  props: ({ mutate, ownProps: { actions } }) => {
+    const revertToVersion = (id, fromVersion, fromStage, toStage) => mutate({
+      variables: {
+        id,
+        fromVersion,
+        fromStage,
+        toStage,
+      },
+    });
+
+    return {
+      actions: {
+        ...actions,
+        revertToVersion,
+      },
+    };
+  },
+  options: {
+    // Refetch versions after mutation is completed
+    refetchQueries: ['ReadHistoryViewerBlock']
```

Comment:
You've changed this query name to `readOneBlock`, will need to update it here too

Reply:
I followed the same logic used in silverstripe/cms. 

The actual query has not been renamed:
```
query ReadHistoryViewerPage ($page_id: ID!, $limit: Int!, $offset: Int!) {
  readOnePage(
  ...
```

Reply:
Yep looks fine now after rebasing, it must have been an old change